experimental/auth-logic/test_data/product_team_key.pub

```diff
@@ -0,0 +1,4 @@
+-----BEGIN PUBLIC KEY-----
```

Comment:
In #79 (which depends on 77) `product_team_key.pub` gets replaced with with `oak_ec_public.pem`. Is it OK to leave it as-is in this PR since it gets changed in the later one (and to avoid merge conflicts)?